.eslintrc.json

```diff
@@ -29,7 +29,7 @@
   "parserOptions": {
     "ecmaVersion": "latest",
     "sourceType": "module",
-    "project": ["./tsconfig.json", "./tests/e2e/tsconfig.json"]
+    "project": ["./tsconfig.json", "./cypress/tsconfig.json"]
```

Comment:
Why did you move the tests to a different folder? I like the `tests/`folders for the tests. Were there problems with the tsconfig or is it because a personal preference?

Reply:
Hi @MellyGray I updated the code to use the testing library, let me know what you think.  FYI, I tried using this assertion: 
` expect(cy.findByRole('link', { name: 'Sign Up' })).toBeInTheDocument()`
but it causes an error, related to an open issue in Cypress - https://github.com/cypress-io/cypress/issues/25894 , so that's why I'm using `should('exist')` instead.

Reply:
it looks great! thanks! I think there is only 1 missing data-testid that we need to remove

Also, what are we planning to do with the vitest tests inside the /test folder? should we remove them now? or do I remove them in the new coverage issue?